plugins/sfIsdiahPlugin/modules/sfIsdiahPlugin/templates/indexSuccess.php

```diff
@@ -73,8 +73,8 @@
 
 <?php end_slot() ?>
 
-<?php if (isset($latitude) && isset($longitude) && null !== $key = sfConfig::get('app_google_maps_api_key')): ?>
-  <div id="front-map" class="simple-map" data-key="<?php echo $key ?>" data-latitude="<?php echo $latitude ?>" data-longitude="<?php echo $longitude ?>"></div>
+<?php if (isset($latitude) && isset($longitude) && null !== $keyProperty = QubitSetting::getByName('google_maps_api_key')): ?>
+  <div id="front-map" class="simple-map" data-key="<?php echo $keyProperty->value ?>" data-latitude="<?php echo $latitude ?>" data-longitude="<?php echo $longitude ?>"></div>
```

Comment:
This is correct but is it necessary?  `lib/filter/QubitSettingsFilter.class.php` loads sfConfig with the settings in `QubitSetting`. In theory `sfConfig::get('app_google_maps_api_key')` should still return the value of `google_maps_api_key` even after the migration to `QubitSetting`. Not 100% sure but give it a shot? Take a look at the filter too if you can!